api/newsfeed/newsfeedModel.js

```diff
@@ -0,0 +1,33 @@
+const db = require('../../data/db-config');
```

Comment:
Looks good I am curious to why everything is being pushed instead of just newsfeed.

Reply:
It's because the other branches have not been merged to main, so the older changes will still be "pushed" to main because the main branch does not have any of those changes from the other PRs.